### PR TITLE
Ilgynoth: Added new option to mark Nightmare Ichor

### PR DESCRIPTION
**Pull request by focusn1k**
Added option for Ilgynoth, allowing to mark the blobs

### CODE REVIEW

File: Nightmare/Ilgynoth.lua [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
 local mod, CL = BigWigs:NewBoss("Il'gynoth", 1094, 1738)
 if not mod then return end
-mod:RegisterEnableMob(105906, 105393, 105304) -- Eye of Il'gynoth, Il'gynoth, Dominator Tentacle
+
+mod:RegisterEnableMob(105906, 105393, 105304, 105721) -- Eye of Il'gynoth, Il'gynoth, Dominator Tentacle, Nightmare Ichor
```
Comment (funkydude) [inline Nightmare/Ilgynoth.lua]: No need for it to be added to enable mobs.

File: Nightmare/Ilgynoth.lua [hunk 2]
```diff
@@ -105,6 +109,9 @@ if L then
 	L.deathglare_tentacle = -13190 -- Deathglare Tentacle
 	L.deathglare_tentacle_icon = 208697 -- Mind Flay icon
 
+	L.nightmare_ichor = -13186
```
Comment (funkydude) [inline Nightmare/Ilgynoth.lua]: I'm not seeing any need for these 2 entries.

**Comment by focusn1k** [comment]
Yes, I did. It works fine with LFR and Mythic difficulty.

File: Nightmare/Ilgynoth.lua [hunk 3]
```diff
@@ -105,6 +109,8 @@ if L then
 	L.deathglare_tentacle = -13190 -- Deathglare Tentacle
 	L.deathglare_tentacle_icon = 208697 -- Mind Flay icon
 
+	L.nightmare_ichor = -13186 -- Nightmare Inchor
```
Comment (funkydude) [inline Nightmare/Ilgynoth.lua]: This can also be removed, there isn't a need for it, use the id directly in the option.

**Comment by funkydude** [comment]
> Yes, I did. It works fine with LFR and Mythic difficulty.

Unfortunately it will not work in conjunction with the tentacle marker option. It is unusual to have 2 icon setting options both trying to run for the entire duration, but it is possible to merge them to work together. Only one function can be registering the target events at one time.

**Pull request closed without merging**